core/eolearn/tests/test_core_tasks.py

```diff
@@ -473,12 +473,27 @@ def test_extract_bands_fails(patch: EOPatch) -> None:
         ExtractBandsTask((FeatureType.DATA, "bands"), (FeatureType.DATA, "EXTRACTED_BANDS"), [2, 4, 16])(patch)
 
 
-def test_create_eopatch():
-    data = np.arange(2 * 3 * 3 * 2).reshape(2, 3, 3, 2)
-    bbox = BBox((5.60, 52.68, 5.75, 52.63), CRS.WGS84)
-
-    patch = CreateEOPatchTask()(data={"bands": data}, bbox=bbox)
-    assert np.array_equal(patch.data["bands"], data)
+@pytest.mark.parametrize(
+    "features, expected",
+    [
+        ({}, 0),
+        ({"data": {"bands": np.arange(0, 32).reshape(1, 4, 4, 2)}}, 1),
+        (
+            {"data": {"bands": np.arange(0, 32).reshape(1, 4, 4, 2), "CLP": np.ones((1, 4, 4, 2))}, "bbox": DUMMY_BBOX},
+            3,
+        ),
+    ],
+)
+def test_create_eopatch(features: Dict[str, Any], expected: int) -> None:
+    created_patch = CreateEOPatchTask()(**features)
+    assert len(created_patch.get_features()) == expected
+
+    for feature in created_patch.get_features():
+        type, name = feature
+        if isinstance(created_patch[feature], np.ndarray):
+            assert_array_equal(created_patch[feature], features[type.value][name])
+        else:
+            assert features[type.value] == created_patch[feature]
```

Comment:
Try to avoid using `type` as a variable name.

Also, here i believe you could do just
```python
assert CreateEOPatchTask()(**features) == EOPatch(**features)
```

I think it's a fair assumption that the interface to `CreateEOPatchTask` and the `EOPatch` constructor are the same.

Reply:
Agree test was testing more `EOPatch()` then `CreateEOPatchTask()`, updated.